packages/quantic/force-app/main/default/lwc/quanticNumberInput/quanticNumberInput.html

```diff
@@ -0,0 +1,45 @@
+<template>
```

Comment:
Should this really be named `quanticNumberInput` ? 
Or is it specifically `quanticNumericFacetFilter` ?